.circleci/scripts/git-diff-develop.ts

```diff
@@ -82,55 +99,69 @@ async function fetchUntilMergeBaseFound() {
  */
 async function gitDiff(): Promise<string> {
   await fetchUntilMergeBaseFound();
-  const { stdout: diffResult } = await exec(`git diff --name-only origin/HEAD...${process.env.CIRCLE_BRANCH}`);
+  const { stdout: diffResult } = await exec(
+    `git diff --name-only origin/HEAD...${SOURCE_BRANCH}`,
+  );
   if (!diffResult) {
-      throw new Error('Unable to get diff after full checkout.');
+    throw new Error('Unable to get diff after full checkout.');
   }
   return diffResult;
 }
 
+function writePrBodyToFile(prBody: string) {
```

Comment:
Q: Why is the PR body written to file? Is there something that prevents it from being passed differently?

Reply:
Good question.  This needs to get to all of the `test-e2e-*` VMs.  I'm trying to avoid calling api.github.com hundreds of times, and instead store it in `artifacts`.